colabs/artifact_basics/Artifact_Basics.ipynb

```diff
@@ -212,18 +213,19 @@
       "outputs": [],
       "source": [
         "run = wandb.init(project=\"artifact-basics\")\n",
-        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to\n",
+        "artifact = run.use_artifact(artifact_or_name=\"my_first_artifact:latest\") # selects the artifact you're adding the file to, and makes a new\n",
         "artifact.add_dir(local_path=\"/content/sample_data\", name=\"new_directory\")\n",
+        "run.log_artifact(artifact, aliases= [\"directory_version\"])\n",
         "run.finish()"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "This adds an entire folder with several .csv files to your Artifact.\n",
+        "This adds a new version of `my_first_artifact` with an entire folder with several .csv files. To save your changes, you'll need to [`log`](https://docs.wandb.ai/ref/python/log) or [`save`](https://docs.wandb.ai/ref/python/artifact#save) them. The directory will overwrite the contents of the previous version of the Artifact.\n",
```

Comment:
I don't think save() can be used for creating a new version - save() is only for properties like description

Reply:
can we make sure the story line here is clear to the user? i.e they updated some files in their directory, they re-add the directory, and we automatically version it. Currently we just add a directory and replace it with another one and it's not clear we're trying to highlight that we automatically version this for you